benchmarks/source-flotiq/gatsby-config.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
```

Comment:
Can you please use `NODE_ENV` in the `dotenv` config as done here? https://github.com/gatsbyjs/gatsby/blob/cf3430462748cecdfb4b9f6b61285534fb4a2ea9/benchmarks/source-contentful/gatsby-config.js#L1